unittests/DOSGuard.cpp

```diff
@@ -91,6 +96,17 @@ TEST_F(DOSGuardTest, Whitelisting)
     EXPECT_FALSE(guard.isWhiteListed(IP));
 }
 
+TEST_F(DOSGuardTest, CIDRWhitelistTest)
```

Comment:
This is not a bad test but **ideally** it belongs in its own test suite in a separate file. Or at least at the bottom of the file like `IntervalSweepHandler` is done atm. 
If you are not afraid to get your hands dirty then feel free to create a separate file for the `IntervalSweepHandler` as well 🥇